packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -203,9 +202,17 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
 
         ...(pointing &&
           (vertical
-            ? pointing === 'end'
-              ? { borderRight: `${pxToRem(3)} solid ${v.primaryActiveBorderColor}` }
-              : { borderLeft: `${pxToRem(3)} solid ${v.primaryActiveBorderColor}` }
+            ? !isFromKeyboard && {
+                '::before': {
+                  content: `''`,
+                  position: 'absolute',
+                  width: pxToRem(3),
+                  height: `calc(100% + ${pxToRem(4)})`,
+                  top: pxToRem(-2),
+                  backgroundColor: v.primaryActiveBorderColor,
+                  ...(pointing === 'end' ? { right: pxToRem(-2) } : { left: pxToRem(-2) }),
+                },
+              }
```

Comment:
We recently merged #1105, it seems that this `:before` creates the same problem.
/cc @miroslavstastny

Reply:
Unfortunately box shadow / border doesn't allow for this kind of visuals.
At least without making the styles grow quite a bit. 

The` :before` element has content set to nothing. Are you sure it's causing problems?

Reply:
Tested it. No problem in this use case. All menu items are read properly.

<img width="1680" alt="Screenshot 2019-03-28 at 13 21 08" src="https://user-images.githubusercontent.com/604667/55154152-c70efc80-515c-11e9-927f-ef237901c849.png">

Reply:
Great, thank you 👍